packages/cozy-doctypes/src/banking/BankingReconciliator.js

```diff
@@ -28,6 +28,14 @@ class BankingReconciliator {
     return { savedAccounts, reconciliatedAccounts }
   }
 
+  /**
+   * Save new accounts and transactions
+   *
+   * @param {array} fetchedAccounts
+   * @param {array} fetchedTransactions
+   * @param  {[type]} options   { logProgress: function }
+   * @returns {[type]} { accounts: savedAccounts, transactions: savedTransactions }
```

Comment:
Please add a proper type instead of "[type]", maybe with a @typedef.

```
    * @param {Array<BankAccount>} fetchedAccounts
    * @param {Array<BankTransactions>} fetchedTransactions

@typedef ReconciliatorResponse
@attribute {Array<BankAccount>} accounts
@attribute {Array<BankTransactions>} transactions
```